cluster/node-pools/master-default/userdata.yaml

```diff
@@ -594,7 +594,7 @@ write_files:
           - --use-service-account-credentials=true
           - --configure-cloud-routes=false
           - --allocate-node-cidrs=true
-          - --cluster-cidr={{ .Cluster.ConfigItems.cluster_cidr}}
+          - --cluster-cidr=10.2.0.0/16
```

Comment:
Should not this be also `10.2.0.0/15` in case of big cluster?

Reply:
Thanks!